packages/server/src/payloads.ts

```diff
@@ -119,7 +119,7 @@ const getArticleEpicTests = async (
             fetchConfiguredArticleEpicHoldbackTestsCached(),
         ]);
 
-        const hardcodedTests = enableHardcodedEpicTests ? epicChoiceCardsTests : [];
+        const hardcodedTests = enableHardcodedEpicTests ? [] : [];
```

Comment:
I guess we can either delete `hardcodedTests` or set `const hardedTests = []`

Reply:
it's nice to keep `enableHardcodedEpicTests` here for the next time

Reply:
@tomrf1 that was my thinking  too. I vote we leave it as it is for now.

Reply:
✅